x-pack/plugins/maps/public/classes/layers/vector_tile_layer/vector_tile_layer.js

```diff
@@ -285,4 +287,9 @@ export class VectorTileLayer extends TileLayer {
   supportsLabelsOnTop() {
     return true;
   }
+
+  async getLicensedFeatures() {
```

Comment:
Is the `vector_tile_layer` module also used by a user adding their own custom vector tile service? Does it make more sense to move this function to the `ems_tms_source` module?

Reply:
`vector_tile_layer` can only be used for EMS. But I will change implementation and push it down to ems_tms_source,  so it's more consistent with how vector_sources work.